### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by TomasTH1**
I hope I finally got it

### CODE REVIEW

File: content/digital-essays/KingsChapel [hunk 1]
```diff
@@ -1,12 +1,18 @@
-**Historic Landmarks at Tremont and School Streets**
+---
+location: Kings-Chapel
```
Comment (garrettdashnelson) [inline content/digital-essays/KingsChapel]: @TomasTH1 the location must be _exactly_ (case-sensitive) `kings-chapel` in order for the site to recognize this essay belongs in that spot.

Comment (TomasTH1) [reply content/digital-essays/KingsChapel]: Do I have to press anything else after renaming it? Or is this still an active pull request that you have to confirm on your end?

**Comment by garrettdashnelson** [comment]
@TomasTH1 your file needs to be named `kings-chapel.md` (with the `.md` extension), otherwise the build software won't recognize it as a Markdown file. Please rename and add to the commit.

**Comment by TomasTH1** [comment]
It appears to work now, thanks.

**Pull request merged**